GameData/Kerbalism/System/Resources.cfg

```diff
@@ -26,6 +26,18 @@ RESOURCE_DEFINITION
   volume = 1
 }
 
+RESOURCE_DEFINITION
+{
+  name = MoistAtmosphere 
+  density = 0.000804          // 1 m^3 of saturated water vapor at STP , 100% humidity
```

Comment:
A specific reason to model it like 100% humidity air? From what I understand this implies conversion from Atmosphere to MoistAtmosphere (including messing with the nitrogen cycle involved with atmosphere generation). WasteAtmosphere is just CO2, why not make this just Water (density)?

Reply:
Kerbalisms Atmosphere, Nitrogen or CO2 has not been used in any of the calculations nor used in any outputs or inputs to the rules or processes used by Humidity, there is no conversion between them.